lumeer-storage/lumeer-storage-mongodb/src/main/java/io/lumeer/storage/mongodb/dao/project/MongoCollectionDao.java

```diff
@@ -70,6 +71,7 @@
 public class MongoCollectionDao extends ProjectScopedDao implements CollectionDao {
 
    private static final String PREFIX = "collections_p-";
+   private static final Random rnd = new Random();
```

Comment:
I think it was for debugging here :)

Reply:
nope, it used to be for a function, but that functionality is removed ;-)